.coveragerc

```diff
@@ -1,10 +1,7 @@
 [run]
 source =
-    rastervision/
+    rastervision
 omit =
-    rastervision/protos/**.py
-    rastervision/utils/filter_geojson.py
-    rastervision/utils/geojson.py
-    rastervision/utils/misc.py
-    rastervision/data/vector_source/label_maker/**.py
-    rastervision/data/label/tfod_utils/**.py
+    */rastervision_core/rastervision/core/data/vector_source/label_maker/*.py
+    */rastervision_core/rastervision/core/data/label/tfod_utils/*.py
+    */rastervision_core/rastervision/core/utils/*.py
```

Comment:
Some of the originally excluded file patterns no longer exist. I've tried to keep the current exclusions in the spirit of the original, ignoring vendored code and non-core utilities. I expect folks more familiar with which code paths are important can tweak this over time if we decide to keep up code coverage reporting.